Source/Core/Core/PowerPC/Jit64/Jit_SystemRegisters.cpp

```diff
@@ -283,7 +283,13 @@ void Jit64::mfspr(UGeckoInstruction inst)
 
 		// An inline implementation of CoreTiming::GetFakeTimeBase, since in timer-heavy games the
 		// cost of calling out to C for this is actually significant.
-		MOV(64, R(RAX), M(&CoreTiming::globalTimer));
+		// Scale downcount by the CPU overclocking factor.
+		CVTSI2SS(XMM0, PPCSTATE(downcount));
+		DIVSS(XMM0, M(&CoreTiming::lastOCFactor));
```

Comment:
It could be worth inverting. @FioraAeterna your thoughts?

We could also detect loops tight loops that only access timebase and jump straight to the correct cycle.